QueAns/admin.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib import admin
+from .models import Question,Meetings,Message,Voting,Satisfaction
+
+
+class MeetingsAdmin(admin.ModelAdmin):
+    def has_change_permission(self, request, obj=None):
+        return (request.method in ['GET', 'HEAD'] and
+                super().has_change_permission(request, obj))
+    def get_readonly_fields(self, request, obj=None):
+        return ['id','venue','moderator','start_time','end_time']
+    def has_add_permission(self, request):
+        if request.user.is_superuser:
+            return True
+    def has_delete_permission(self, request, obj=None):
+        return False
+
+
+class QuestionAdmin(admin.ModelAdmin):
+    fields = ['question_text','anon_status','meeting_id',]
+    list_filter = ['meeting_id',]
+
+    def save_model(self, request, obj, form, change):
+        obj.asker = request.user
+        obj.save()
```

Comment:
When we override a method, we should always call `super()` if we need to replicate behavior done by original method.
Why?
- What would happen here if the implementation of the original method changes?


Also, write a docstring to this method telling why did we override this method.

Reply:
Okay, Done.